client/src/components/modals/pick-entity-or-conversion.vue

```diff
@@ -138,7 +133,7 @@ export default class PickEntityOrConversion extends Vue {
     }
     let cols = this.tableData.length
     // 210 per column with a max of 900px
-    return `${210 * cols > 900 ? 900 : 210 * cols}px`
+    return `${210 * cols > 900 ? 960 : 210 * cols}px`
```

Comment:
Should this be
`if X is greater than Y : Y : X`

(iow, first check should be if > 960) ?

Reply:
Going to have another look at this, i think we might need set a larger minimum. Some dialogs with only 1-2 cells are pretty tight. Stay tuned!

Reply:
While you're there, it would be great to use existing breakpoints (this.$breakpoint)!

Reply:
updated!